lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/YarnClusterSubmissionFromCS.java

```diff
@@ -89,6 +92,13 @@ private YarnClusterSubmissionFromCS(final AvroYarnAppSubmissionParameters yarnAp
     this.yarnDriverStdoutFilePath = yarnClusterJobSubmissionParameters.getDriverStdoutFilePath().toString();
     this.yarnDriverStderrFilePath = yarnClusterJobSubmissionParameters.getDriverStderrFilePath().toString();
 
+    if (yarnClusterJobSubmissionParameters.getEnvMap() != null) {
+      for (Map.Entry<java.lang.CharSequence, java.lang.CharSequence> pair :
+          yarnClusterJobSubmissionParameters.getEnvMap().entrySet()) {
+        this.envMap.put(pair.getKey().toString(), pair.getValue().toString());
```

Comment:
There must be a utility function in Apache Commons to do this :)

Reply:
Didn't find good one. Do you know any?